driver/config/config.go

```diff
@@ -1074,6 +1076,14 @@ func (p *Config) SelfServiceLinkMethodBaseURL() *url.URL {
 	return p.p.RequestURIF(ViperKeyLinkBaseURL, p.SelfPublicURL())
 }
 
+func (p *Config) DatabaseCleanupSleepTables() time.Duration {
+	return p.p.DurationF(ViperKeyDatabaseCleanupSleepTables, 5*time.Second)
```

Comment:
Same here, the default should be provided by the schema.
```suggestion
	return p.p.Duration(ViperKeyDatabaseCleanupSleepTables)
```